models/book.js

```diff
@@ -0,0 +1,15 @@
+'use strict'
+const mongoose = require('mongoose');
+const { Schema } = mongoose;
+
+const bookSchema = new Schema({
+  title: String,
+  genre: String,
+  description: String,
+  checkedIn: Boolean,
+});
+
+
+const Book = mongoose.model('Book', bookSchema);
+
+module.exports = Book;
```

Comment:
Good work here! I would consider renaming file to be bookSchema or something similar so it is clear exactly what this file holds. 👍

Reply:
- Build a Mongoose 'Book' schema with valid keys for **title, description, status, and email (**of the user that added the Book).